Data-Engineering/Airflow/example_ezaf_spark_csv_to_parquet_fts.yaml

```
@@ -4,37 +4,35 @@ metadata:
   name: spark-fts-{{ts_nodash|replace("T", "")}}
 spec:
   sparkConf:
-    spark.mapr.extraconf.secret: {{dag_run.conf.get("s3_secret_name", "spark-s3-creds")}}
     spark.hadoop.fs.s3a.path.style.access: "true"
     spark.hadoop.fs.s3a.impl: "org.apache.hadoop.fs.s3a.S3AFileSystem"
     spark.hadoop.fs.s3a.connection.ssl.enabled: "true"
+    spark.mapr.extraconf.secret: "<k8s-secret-name>" 
```

Let's use template here and in `Data-Engineering/Airflow/example_ezaf_spark_csv_to_parquet_fts.py` we can return back `'s3_secret_name': Param("spark-s3-creds", type="string"),` and remove `jwt_token` parameter in such case. I think it's relevant to any Airflow DAG
```suggestion
    spark.mapr.extraconf.secret: {{dag_run.conf.get("s3_secret_name", "spark-s3-creds")}}  
```